django/db/backends/mysql/operations.py

```diff
@@ -456,8 +456,5 @@ def on_conflict_suffix_sql(self, fields, on_conflict, update_fields, unique_fiel
             )
             return conflict_suffix_sql % {"fields": fields}
         return super().on_conflict_suffix_sql(
-            fields,
-            on_conflict,
-            update_fields,
-            unique_fields,
+            fields, on_conflict, update_fields, unique_fields
```

Comment:
Please revert all unrelated reformatting and new blank lines before you will ask for another review.